agenthub/codeact_agent/action_parser.py

```diff
@@ -42,6 +50,8 @@ def parse_response(self, response) -> str:
         for lang in ['bash', 'ipython', 'browse']:
             if f'<execute_{lang}>' in action and f'</execute_{lang}>' not in action:
                 action += f'</execute_{lang}>'
+            if '<file_edit' in action and '</file_edit>' not in action:
+                action += '</file_edit>'
```

Comment:
```suggestion
        if '<file_edit' in action and '</file_edit>' not in action:
            action += '</file_edit>'
```
Don't need to detect file_edit for every language.